core/src/main/scala/kafka/network/RequestChannel.scala

```diff
@@ -73,14 +73,14 @@ object RequestChannel extends Logging {
     buffer = null
     private val requestLogger = Logger.getLogger("kafka.request.logger")
 
-    private def requestDesc: String = {
+    private def requestDesc(details: Boolean): String = {
       if (requestObj != null)
-        requestObj.describe(false)
+        requestObj.describe(details)
       else
         header.toString + " -- " + body.toString
```

Comment:
I think we need a followup JIRA to add a trace/debug description modes to the new Request/Response API and apply the same logic here. Or even do it in this JIRA.

Otherwise, in few weeks when most requests are using the new objects, we are back to square one.